types/bech32/bech32.go

```diff
@@ -0,0 +1,32 @@
+package bech32
```

Comment:
Why creating a new `types` sub directory for just two functions? Can't we just drop such functions in `types`?

Reply:
up to you guys. I put it in a subdirectory because its not a type but util/lib and there doesnt seem to be a place for these to live.